pkg/binfmt/Makefile

```diff
@@ -23,6 +23,12 @@ push: tag
 	rm -f hash
 	docker rmi $(IMAGE):build || true
 
```

Comment:
this Makefile doesn't `pull` the base image, so is missing the trust check on that